FurnitureProject/FurnitureProject/Common/UserManagerExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+using Microsoft.AspNet.Identity;
+using FurnitureProject.Models;
+
+namespace FurnitureProject.Common
+{
+    public static class UserManagerExtensions
+    {
+        public static bool ChangeUserAvatar(this ApplicationUserManager UserManager, string UserID, string ImagePath)
+        {
+            using(ApplicationDbContext context = new ApplicationDbContext())
```

Comment:
This is ambiguous, since we can retrieve the user with the method GetUser() below